taca_ngi_pipeline/deliver/deliver_grus.py

```diff
@@ -463,28 +462,31 @@ def _set_pi_details(self, given_pi_email=None):
             logger.error("Cannot fetch PI SNIC id using snic API. Error says: {}".format(str(e)))
             raise e
     
-    def _set_binfo_details(self):
+    def _set_other_member_details(self, other_member_emails=[]):
         """
-            Set bioinfo contact details if avilable, this is not mandatory so
+            Set other contact details if avilable, this is not mandatory so
             the method will not raise error if it could not find any contact
         """
-        self.binfo_email, self.binfo_snic_id = (None, None)
-        # try getting bioinfo contact email
+        self.other_member_snic_ids = []
+        # try getting appropriate contact emails
         try:
-            self.binfo_email = self._get_order_detail()['project_bx_email']
-            assert self.binfo_email #check its not empty string
-            logger.info("Bioinfo contact for project {} found: {}".format(self.projectid, self.binfo_email))
+            prj_order = self._get_order_detail()
+            owner_email = prj_order.get('owner', {}).get('email')
+            if owner_email and owner_email != self.pi_email and owner_email not in other_member_emails:
+                other_member_emails.append(owner_email)
```

Comment:
not sure that I agree that you should add the primary contact email by default at this point (unless that is something you have decided to always do). I would suggest passing a flag on the command line or having a setting in the config that can moderate whether to include this contact or not. That way, you could still deploy the code but just skip adding this contact.